internal/flink/command_artifact_create.go

```diff
@@ -103,3 +120,35 @@ func (c *command) createArtifact(cmd *cobra.Command, args []string) error {
 	})
 	return table.Print()
 }
+
+func (c *command) validateCreateArtifact(cmd *cobra.Command, args []string) error {
```

Comment:
is this really necessary? cli is already validate file name line 61 + plugin type is also validated at server side

Reply:
This function gets executed as pre check, so we can avoid going to request phase. And now as we are introducing runtime-language there might be case that user might specify `.jar` and runtime-lang as `python`. This function also helps with such validation, and we can avoid request round trip. But it's good to have if you think it's un-necessary I can remove it.

Reply:
i don't think we need this. but i'll let CLI team to confirm.

Reply:
in general, the benefits is if anything changes, it can be easily update on server side without changing CLI.

Reply:
This is good point hmm 🤔 . Yeah am having second thoughts now @brianstrauch  do you have any suggestions ?

Reply:
Thank you @byang-confluent  and @brianstrauch  I was overthinking it 😅 so I wrote all validation but this ^ makes more sense. I removed function. Restored previous check (refactor it a bit to allow zip) and added check for runtime language let me know if that's also un necessary.